.github/workflows/code-checks.yml

```
@@ -0,0 +1,59 @@
+name: Code Checks
+
+on:
+  pull_request:
+    branches:
+      - main
```

`pg-to-dbml` was prior to the Github rollout of the new main branch name.

```suggestion
      - master
```